tests/test_tztrout.py

```diff
@@ -0,0 +1,433 @@
+            assert tz_id in ids
+
+    @pytest.mark.parametrize(
+        'state, city, phones, tz_name',
+        [
+            # NSW - New South Wales
+            # WA - Western Australia
+            # NT - Northern Territory
+            # SA - South Australia
+            # TAS - Tasmania
+            # VIC - Victoria
+            # ACT - Australian Capital Territory
+            # QLD - Queensland
+            (
+                'NSW',
+                'Sydney',
+                ['+61 27 333 4444', '+61 28 333 4444', '+61 29 333 4444'],
+                'AET',
+            ),
+            (
+                'WA',
+                'Perth',
+                [
+                    '+61 852 22 4444',
+                    '+61 853 22 4444',
+                    '+61 854 22 4444',
+                    '+61 861 22 4444',
+                    '+61 862 22 4444',
+                    '+61 863 22 4444',
+                    '+61 864 22 4444',
+                    '+61 865 22 4444',
+                ],
+                'AWT',
+            ),
+            ('NT', 'Darwin', ['+61 879 22 4444', '+61 889 22 4444'], 'ACT'),
+            (
+                'SA',
+                'Adelaide',
+                [
+                    '+61 870 22 4444',
+                    '+61 871 22 4444',
+                    '+61 872 22 4444',
+                    '+61 873 22 4444',
+                    '+61 874 22 4444',
+                    '+61 881 22 4444',
+                    '+61 882 22 4444',
+                    '+61 883 22 4444',
+                    '+61 884 22 4444',
+                ],
+                'ACT',
+            ),
+            ('TAS', 'Hobart', ['+61 361 22 4444', '+61 362 22 4444'], 'AET'),
+            (
+                'VIC',
+                'Melbourne',
+                ['+61 37 333 4444', '+61 38 333 4444', '+61 39 333 4444'],
+                'AET',
+            ),
+            (
+                'ACT',
+                'Canberra',
+                [
+                    '+61 251 22 4444',
+                    '+61 252 22 4444',
+                    '+61 261 22 4444',
+                    '+61 262 22 4444',
+                ],
+                'AET',
+            ),
+            ('QLD', 'Brisbane', ['+61 72 333 4444', '+61 73 333 4444'], 'AET'),
+            (
+                'QLD',
+                'Townsville',
+                ['+61 744 22 4444', '+61 745 22 4444', '+61 777 22 4444'],
+                'AET',
+            ),
+        ],
+    )
+    def test_major_cities_australia(
+        self, state, city, phones, tz_name, au_tz_names
+    ):
+        """ Make sure all the major cities in Australia match the right time zone
+        (and the right time zone only). """
+        ids = tztrout.tz_ids_for_address('AU', state=state, city=city)
+        for phone in phones:
+            ids2 = tztrout.tz_ids_for_phone(phone)
+            assert ids == ids2
+        self.assert_only_one_tz(ids, tz_name, au_tz_names)
```

Comment:
Again, we're testing two different things here. Doesn't make sense to group them together IMO.